discovery-provider/src/schema_track_simple.json

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
I would move this files to src/schemas/schema_track.json and the other one to the same dir